.github/workflows/release.yaml

```diff
@@ -29,3 +29,17 @@ jobs:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GH_RELEASE_TOKEN }}
+      - name: Set up Java
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
```

Comment:
For the Java version, [A Java version between 8 and 17 is required to execute Gradle.](https://docs.gradle.org/current/userguide/compatibility.html#:~:text=A%20Java%20version%20between%208,compilation%20and%20forked%20test%20execution.)
I would suggest we stick with version 11 for now, because it is established and a LTS version. But, any version between those should work.